README.md

```diff
@@ -13,19 +13,30 @@ In 2019 we create [Undebates](https://github.com/EnCiv/undebate) and launched th
 
 # Getting Started
 
-The following has been tested using node v16.13.0 and npm v8.1.0 on Windows 10 and MacOs.  There are know issues using nmp < 7. During install, you can ignore the warning about incompatibility with previous previous versions of node/npm - we need to go back to those packages and update their dependencies.
+The following has been tested using node v16.13.0 and npm v8.1.0 on Windows 10 and MacOs.  There are known issues using npm < 7. During install, you can ignore the warning about incompatibility with previous versions of node/npm - we need to go back to those packages and update their dependencies.
 
+Instructions for cloning and running storybook on your local system using your CLI of choice is as follows: 
 ```
-git clone https://github.com/EnCiv/undebate-ssp.git
+// Create a directory on your local system to hold the project that you will be cloning (e.g., 'undebate-ssp')
+mkdir undebate-ssp  
```

Comment:
Can you check this. I think that this would create ... /undebate-ssp/undebate-ssp
git clone will create a directory and put the project in there, unless you add a second parameter to say where to put it.
So you don't need the mkdir, and the cd undebate-ssp should come after the git clone

Reply:
It's a good idea to add the comments but they should begin with # because these are shell commands - // is javascript.  It's a detail, but sometimes these things get cut and pasted.

Reply:
Thanks. I fixed that now they are #'s.